pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -121,8 +121,12 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 			}
 		}
 	}
-	if dc.BootLoader == "" && (dc.VirtualizationModeOrDefault() == types.FML || runtime.GOARCH == "arm64") {
-		dc.BootLoader = "/usr/lib/xen/boot/OVMF_CODE.fd"
+	if dc.BootLoader == "" {
+		if dc.VirtualizationModeOrDefault() == types.FML {
+			dc.BootLoader = "/usr/lib/xen/boot/OVMF_CODE.fd"
+		} else if runtime.GOARCH == "arm64" {
+			dc.BootLoader = "/usr/lib/xen/boot/ovmf.bin"
```

Comment:
this is the same path as https://github.com/lf-edge/eve/pull/4327/files#diff-de84b12c2ce63e5d98657e2297135ce3af53221cbb4611fa7af9534c9e3a7e98R120 isn't it? Perhaps use a constant for it?

Reply:
Thx. Resolved.